tox.ini

```diff
@@ -1,12 +1,19 @@
 [tox]
-envlist = py27, py35, py36, py37, flake8
+envlist = py38, py39
 skip_missing_interpreters = true
 
 [testenv]
 deps=
-  pytest
+    -rrequirements.txt
+    -rrequirements-test.txt
 
-commands=pytest --junitxml=.tox/results/tox/junit-{envname}.xml {posargs:tests} --ignore=tests/functional
+passenv=ANCHORE_* DOCKER_* NO_PROXY TEST_VULNERABILITIES_PROVIDER
+
+commands=
```

Comment:
For my own knowledge, why has the commands block changed here?

Reply:
Also, is the --ignore=tests/functional no longer needed?

Reply:
The commands block changed to add code coverage reporting like we have in engine and enterprise. 

I removed the --ignore=tests/functional because I didn't believe it was doing anything.  If you looks at the CI jobs it was running the functional tests anyway.  https://app.circleci.com/pipelines/github/anchore/anchore-cli

Reply:
Ack, thanks